Gems/Atom/Feature/Common/Assets/ShaderResourceGroups/Stars/SceneSrg.azsli

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Super nit for this submission: I think you mean 'Planet' instead of star in all of the references, however, you probably did this by design since 'star' is more easily understood.